Exception in thread "main" java.lang.reflect.UndeclaredThrowableException
at $Proxy0.display(Unknown Source)
at proxy.ProxyApp.createAndTestProxy(ProxyApp.java:23)
at proxy.ProxyApp.main(ProxyApp.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
at java.lang.reflect.Method.invoke(Method.java:597)
at com.intellij.rt.execution.application.AppMain.main(AppMain.java:110)
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
at java.lang.reflect.Method.invoke(Method.java:597)
at proxy.ProxyHandler.invoke(ProxyHandler.java:26)
... 8 more
Caused by: java.sql.SQLException
at proxy.ClassA.display(ClassA.java:18)
... 13 more
